eng/pipelines/templates/jobs/sdk-build.yml

```diff
@@ -49,10 +49,18 @@ jobs:
     enableSbom: ${{ parameters.enableSbom }}
     variables:
     - ${{ insert }}: ${{ parameters.variables }}
+    templateContext:
+      sdl:
+        binskim:
+          analyzeTargetGlob: +:f|artifacts\bin\**\*.dll;+:f|artifacts\bin\**\*.exe;-:f|artifacts\bin\**\msdia140.dll;
```

Comment:
Good find. I didn't know the template context can override the general template parameters.